_includes/footer.html

```diff
@@ -1,14 +1,20 @@
 <footer class="footer">
   <div class="container">
     <div class="columns">
-      <div class="column">
+      
+      <div class="column" style="text-align: center;">
+        <img src="https://avatars.githubusercontent.com/u/25749259?s=150&amp;v=4" alt="Drawpile Logo"><br>
         <a href="https://github.com/drawpile/" target="_blank"><span class="icon"><span class="fab fa-github"></span></span> Github</a><br>
         <a href="https://discord.gg/M3yyMpC" target="_blank"><span class="icon"><span class="fab fa-discord"></span></span> Discord</a><br>
-        <a href="/devblog/" target="_blank"><span class="icon"><i class="fa fa-code"></i></span> Development Blog</a><br>
+        <a href="/devblog/" target="_blank"><span class="icon"><i class="fa fa-code"></i></span> Dev Blog</a><br>
       </div>
-      <div class="column is-narrow">
-        <p>&copy; 2014-2023 Drawpile Contributors</p>
-        <p>The content of this site (unless otherwise noted) is licensed <a href="https://creativecommons.org/licenses/by-sa/4.0/">CC-SA 4.0</a>.</p>
+      <div class="column">
+
+        
+      </div><div style="display: flex; flex-direction: column; justify-content: center; row-gap: 1rem;" class="column is-narrow">
```

Comment:
Don't interfere with Bulma's layout like this without a good reason. Laying out some text is not a good reason. Also don't use `style`, put extra styling into `_sass/main.scss` instead.